### ISSUE TITLE
Three Missing Icons in UI (Windows)

### ISSUE THREAD

**Issue by WildByDesign** [bleachbit/bleachbit#644]
BleachBit 2.3 Beta
Windows 10 64-bit

Regarding the following commit: 
**Fix: three icons were missing**
[https://github.com/bleachbit/bleachbit/commit/bfd5b0fd05a4312c4a1faf3e3d09eb0471af023a](url)

Unfortunately, I must confirm that this fix did not fix the problem with the missing icons. I have tried several of the daily/unstable builds since that commit and now also the 2.3 Beta that you released today, and all of those builds are still missing the icons in the UI on Windows 10.

![BleachBit-missing-icons](https://user-images.githubusercontent.com/26308319/66445759-e644eb80-ea15-11e9-8b8b-c8a01ea4f628.png)

Missing Icons: Clean, Stop and Restore Down

Note: The Restore Down button is only missing when BleachBit UI is maximized full screen. The Clean and Stop buttons are missing at all times.

Thank you for your time. Please let me know if there is a fix available and I will be happy to test and provide feedback.

**Comment by az0** [comment]
Thanks for the report. Please try build [2.3.1303](https://ci.bleachbit.org). It is building now.

**Comment by WildByDesign** [comment]
You're welcome. Thank you for the fast response and quick commit to this issue.

It looks like the Clean icon and the Restore Down icons are working and look fantastic. However, it looks like the Abort/Stop icon is still missing.

**Comment by WildByDesign** [comment]
![BleachBit-missing-icon](https://user-images.githubusercontent.com/26308319/66448803-c666f500-ea20-11e9-9c27-1a8f9e74b781.png)

This is how it looks on Windows 10 after the 2.3.1303 build. I assume the blank look of the Abort/Stop button indicates that the icon is still missing for only that one button now.

**Comment by az0** [comment]
Thanks for the feedback. Yes, there's still one missing icon. I'll check
again.

**Comment by WildByDesign** [comment]
I found an appropriate fix for this.

I grabbed the file _process-stop-symbolic.svg_ from a previous build that was built prior to the icon stripping occurred. I grabbed from build 2.3.1154.

I copied that file to the usual directory for svg icons: _icons\Adwaita\scalable\actions_

Interestingly, it failed to load. I had to renamed the icon from _process-stop-symbolic.svg_ to _process-stop.svg_ and now the icon is working.

![BleachBit-missing-icon-fixed](https://user-images.githubusercontent.com/26308319/66450336-1d22fd80-ea26-11e9-8266-d75f244375f4.png)

Now all of the action button icons are proper symbolic icons and look great.

**Comment by az0** [comment]
That's interesting about the renaming.

For build 2.3.1304, I whitelisted `process-stop-symbolic.svg` and `process-stop.png`. In my test, the stop icon now works. (I did not rename any icons.)

**Comment by WildByDesign** [comment]
Thank you, Andrew. You are right, all of the icons are working now without the need to rename them. I decided to start with a fresh bleachbit.ini config to confirm that and everything is working now as expected.

I have only one suggestion / nitpick:

All of the button icons are showing colored designs, however, the Clean (clear all) button is showing the symbolic icon instead of color. I would suggest that the icons match, whether that be all color or all symbolic. Either way, as long as they all match.

Current (2.3.1304):
![BleachBit-icon-2 3 1304](https://user-images.githubusercontent.com/26308319/66485251-7d419000-ea76-11e9-9c92-14825a3016cb.png) 
* _edit-clear-all-symbolic.svg_ is being used here (above) and should be colored icon

Symbolic (example):
![BleachBit-missing-icon-fixed](https://user-images.githubusercontent.com/26308319/66485590-15d81000-ea77-11e9-9ad5-b376a7acb0fc.png)

I simply renamed a couple of icons to achieve the Symbolic working example and am currently using that right now. I find that the Symbolic icons look best with the Dark theme and the colored icons seem to work best with the Light theme.

Thank you for your time and for sharing your great work with the community. Cheers!
